src/tests/utils/url-to-html.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { linkExists, extractUrl, replaceLinks, convertArrayToObject } from "../../app/utils/status.util";
+
+describe('url should', () => {
```

Comment:
It's a best practise to put the test suit name here, something like `URL TestSuite`